zebra-consensus/src/lib.rs

```diff
@@ -16,3 +16,32 @@
 pub mod checkpoint;
 pub mod mempool;
 pub mod verify;
+
+/// Test utility functions
+///
+/// Submodules have their own specific tests.
+#[cfg(test)]
+mod tests {
+    use std::sync::Once;
+    use tracing_error::ErrorLayer;
+    use tracing_subscriber::prelude::*;
+    use tracing_subscriber::{fmt, EnvFilter};
+
+    static LOGGER_INIT: Once = Once::new();
+
+    // TODO(jlusby): Refactor into the zebra-test crate (#515)
+    pub(crate) fn install_tracing() {
+        LOGGER_INIT.call_once(|| {
+            let fmt_layer = fmt::layer().with_target(false);
+            let filter_layer = EnvFilter::try_from_default_env()
+                .or_else(|_| EnvFilter::try_new("info"))
+                .unwrap();
+
+            tracing_subscriber::registry()
+                .with(filter_layer)
+                .with(fmt_layer)
+                .with(ErrorLayer::default())
+                .init();
+        })
+    }
+}
```

Comment:
This is merged into `main` as of a few minutes ago so it can now be replaced with a call to `zebra_test::init`.

Reply:
I think I'll fix this issue in #538, then rebase this PR.

Reply:
Hmm, actually, that's going to cause some nasty rebase conflicts. Let's do this change in this PR.

Reply:
Done in ed574d540e98734443666165de70cfd7aaaf5e76.